.github/workflows/proof_ci_resources/config.yaml

```diff
@@ -1,5 +1,5 @@
 cadical-tag: latest
-cbmc-version: latest
+cbmc-version: "5.95.1"
```

Comment:
Why is this necessary? This means that our cbmc version will get out of date, since there's no mechanism to trigger us to bump this version.

Reply:
CBMC just released version 6, which contains breaking changes, so we need to pin this version temporarily to unblock the CI. They plan to issue follow-up PR that will return s2n-tls to using CBMC's latest release: https://github.com/aws/s2n-tls/pull/4586

Reply:
Oh, I see. The version change is unrelated to this PR. CBMC is currently failing for all PRs.

In that case, can you open an issue to remind us we need to bump the version back to latest? I don't us to lose track of that problem and never get the work done.

In the future (and maybe in this case), it's a good idea to open a separate PR for high priority / broken CI issues like that unrelated to your PR. That way all open PRs aren't blocked on your unrelated changes.

Reply:
Noted!

Reply:
> In the future (and maybe in this case), it's a good idea to open a separate PR for high priority / broken CI issues like that unrelated to your PR. That way all open PRs aren't blocked on your unrelated changes. That's also good in the hypothetical case that we have to revert your changes -- we wouldn't want to also re-break the CI.

Wasn't #4586 doing exactly that?